uwblueprint/planet-read
# Add level and language filters to admin table

## Notion ticket link



[Create Translators/Reviewers filters](https://www.notion.so/uwblueprintexecs/Create-Translators-Reviewers-filters-d065266431124e27a44e595bb91a6c4f)



## Implementation description

- added filtering on frontend for the users table for admin



## Steps to test

1. login as an admin (Angela Merkel)
2. go to http://localhost:3000/admin 
3. try adding filters (level, language, search) --> Note: adding a level filter without a language filter won't do anything 
4. check that filters apply correctly
5. check that dropdowns are disabled if an option is selected
6. check that badges appear and are clearable 



## What should reviewers focus on?

- do the filters work
- do the dropdowns work
- do the badges work
- does it look right 


## Checklist
